gen/field.go

```diff
@@ -30,6 +30,7 @@ var reservedIdentifiers = map[string]struct{}{
 	"ToWire":   {},
 	"FromWire": {},
 	"String":   {},
+	"Equals":   {},
```

Comment:
Can you please point me where similar checks are done for toWire/fromWire. I can add code for equals too there.